File: Data Structures & Algorithms/contains-duplicate-ii/submission-1.java
class Solution {
    public boolean containsNearbyDuplicate(int[] nums, int k) {
        int l =0 ,r =1;
        while(r<nums.length){
            if(nums[l]==nums[r]){
                if(r-l <= k){
                    // System.out.println(nums[l] + " ->"+nums[r] + " , :" + (r-l) );
                    return true;
                }
                l++;
            }
            r++;
        }
        return false;
    }
}

File: Data Structures & Algorithms/missing-number/submission-1.py
class Solution:
    def missingNumber(self, nums: List[int]) -> int:
        # res=len(nums)
        # for i in range(len(nums)):
        #     res+=i-nums[i]
        # return res
        for i in range(len(nums)):
            if i not in nums:
                return i
        